demos/defender/defender_demo_json/mqtt_operations.c

```diff
@@ -867,17 +992,9 @@ bool SubscribeToTopic( const char * pTopicFilter,
          * of receiving publish message before subscribe ack is zero; but application
          * must be ready to receive any packet. This demo uses MQTT_ProcessLoop to
          * receive packet from network. */
```

Comment:
This is a nit, but probably should remove the last sentence of this comment? That or expand upon how pvWaitForPacketAck() is doing this?